sites/fast-creator/app/configs/library.fast.design-system.mapping.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    accentPalette,
+    baseLayerLuminance,
+    cornerRadius,
+    disabledOpacity,
+    fillColor,
+    focusOutlineWidth,
+    outlineWidth,
+    PaletteRGB,
+    SwatchRGB,
+} from "@microsoft/fast-components";
+import { parseColorHexRGB } from "@microsoft/fast-colors";
+
+export function mapFASTComponentsDesignSystem(designSystem: {
+    [key: string]: any;
+}): void {
+    Object.entries(designSystem).forEach(([attribute, value]: [string, any]) => {
+        switch (attribute) {
+            case "fill-color":
```

Comment:
Is there any way these can come from the design token declaration `name` in case they ever change?

Reply:
FWIW the design tokens themselves have a 'name' property that could be used, though there is no such token "accent-color"

Reply:
Some of these are obfuscating what is actually happening under the hood since a 1 to 1 UX pattern with design tokens in this case is undesirable.